src/ship/passiveSensors.js

```diff
@@ -1,5 +1,6 @@
-import Vector2 from "../helpers/Vector2.js";
 import PassiveSensorReading from "./passiveSensorReading.js";
+import response from "../helpers/response.js";
+
 export default class PassiveSensors{
     constructor(parentShip){
 		this.parentShip = parentShip;
```

Comment:
Same as first comment. Return `Response` object with error for missing solar system (first line of `generatePassiveSensorReadings`).